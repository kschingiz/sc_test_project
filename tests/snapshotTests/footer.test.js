import React from "react";
import Footer from "../../components/footer";
import renderer from "react-test-renderer";

it("Footer renders correctly", () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
