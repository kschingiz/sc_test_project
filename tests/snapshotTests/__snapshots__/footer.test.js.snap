// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="makeStyles-footer-1"
>
  <div>
    About Us Privacy & Cookies Terms
  </div>
  <div>
    Follow Us: Social network links
  </div>
  <div>
    Made with love in San Francisco
  </div>
</div>
`;
